unit/test_set_control_instance.py

```diff
@@ -6,6 +6,20 @@
 from library.cartridge_set_control_instance import get_control_instance
 
 
+twophase_commit_versions = {}
+
+
+def get_twophase_commit_versions_mock(_, advertise_uris):
+    versions = [
+        twophase_commit_versions.get(uri, 1) for uri in advertise_uris
+    ]
+    return versions, None
+
+
+import library.cartridge_set_control_instance as lib
```

Comment:
Let's move this import to other imports

Reply:
This import is used only for mocking function, I think that it's ok to place this logic separately.

Reply:
It's better to do all the imports at the top anyway:
https://www.flake8rules.com/rules/E402.html